ocos/kernels/kernels.h

```diff
@@ -11,11 +11,13 @@ typedef CPTR_OrtCustomOp (*FxGetSchemaInstance)();
 FxGetSchemaInstance const* GetCustomOpSchemaList();
 
 struct BaseKernel {
-  BaseKernel(OrtApi api) : api_(api), ort_(api_) {}
+  BaseKernel(OrtApi api) : api_(api), info_(nullptr), ort_(api_) {}
+  BaseKernel(OrtApi api, const OrtKernelInfo *info) : api_(api), info_(info), ort_(api_) {}
 
  protected:
   OrtApi api_;  // keep a copy of the struct, whose ref is used in the ort_
   Ort::CustomOpApi ort_;
+  const OrtKernelInfo* info_;
```

Comment:
where was 'info_' used?

Reply:
I reused this implementation in PR #41, info_ is needed to retrieve parameters.